Get_Started/login.css

```diff
@@ -0,0 +1,62 @@
+.login-page{
+    display: flex;
+    justify-content: center;
+}
+
+.login-container{
+    margin-top: 2rem;
+    color: var(--primary-color);
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: 40rem;
+    height: 28rem;
+    background-color: rgb(255, 255, 255);
+    border: 1px solid #ff7e71;
+    border-radius: 5px;
+}
+
+.margin-bottom{
+    margin-bottom: 1rem;
+}
+
+.login-elements{
+    display: flex;
+    width: 23.5rem;
+    align-items: center;
+    justify-content: space-between;
+    padding-bottom: 2rem;
+}
+
+.element-checkbox{
+    display: flex;
+    align-items: center;
+    gap: 0.5rem;
+}
+
+#checkbox{
+    width: 16px;
+    height: 16px;
```

Comment:
use em or rem . 16px -> 1rem . change this. For width and height use - vw and vh .